src/composables/multisigAccountCreate.ts

```diff
@@ -163,17 +182,43 @@ export function useMultisigAccountCreate({ store }: IDefaultComposableOptions) {
       multisigAccountCreationPhase.value = MULTISIG_CREATION_PHASES.deployed;
       const gaContract = await sdk.getAccount(accountId);
       multisigAccountCreationPhase.value = MULTISIG_CREATION_PHASES.created;
+
       multisigAccount.value = {
         contractId: gaContract.contractId,
-        multisigAccountId: accountId,
+        balance: new BigNumber(gaContract.balance),
+        gaAccountId: accountId,
+        signers,
+        confirmationsRequired,
+        consensusLabel: prepareConsensusLabel(0, confirmationsRequired, signers.length),
+        nonce: 1, // Default value for freshly created account
+        createdAt: dayjs().toISOString(),
+        updatedAt: dayjs().toISOString(),
+        hasPendingTransaction: false,
+        confirmedBy: [],
+        expired: false,
+        id: 0,
+        proposedBy: '',
+        refusedBy: [],
+        txHash: undefined, // set from propose
+        version: SUPPORTED_MULTISIG_CONTRACT_VERSION,
+        expirationHeight: 0,
+        signerId: signers[0],
+        height: -1,
+        pending: true,
       };
+
       delete pendingMultisigCreationTxs.value[accountId];
     } else {
       throw Error('Vault creation transaction is not mined within the expected time');
     }
 
-    // Wait for the account to be discovered by the wallet to allow to open it's details.
-    // TODO in the future we could try to add the new account to the list of the accounts.
+    addPendingMultisigAccount(multisigAccount.value);
+    setActiveMultisigAccountId(multisigAccount.value.gaAccountId);
+
+    const delay = () => new Promise(() => setTimeout(() => {}, 10000));
+
+    await delay();
```

Comment:
This looks rather bad. At least add comment that would describe why we need to wait 10s. Also the delay function should be moved to utils in my opinion.

Reply:
Ooops, forgot to delete it